perf-dashboard/example-data.json

```diff
@@ -0,0 +1,161 @@
+{
+    "$schema": "./benchmarks.schema.json",
+    "benchmarks": [
+        {
+            "name": "Single Connection throughput – Upload",
+            "unit": "bits/s",
+            "comparisons": [
+                {
+                    "name": "http",
```

Comment:
I assume https instead of http?

Also, do we want to be more clear than just "https"?  Should we specify whether this is using go stdlibrary, rust stdlibrary, curl, etc?  (and if so, I assume the version of go, rust, curl, etc. matters).  At which point is a "comparison" very different than a result?